apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nav-sidebar/src/components/nav-sidebar/index.tsx

```diff
@@ -280,20 +256,33 @@ function useNavItems(): Post[] {
 				status: statusFilter,
 			} ) || [];
 
-		const currentPost = {
+		const current = {
 			id: currentPostId,
 			slug: getEditedPostAttribute( 'slug' ),
 			status: isEditedPostNew() ? 'draft' : getEditedPostAttribute( 'status' ),
 			title: { raw: getEditedPostAttribute( 'title' ), rendered: '' },
 		};
 
-		return [ currentPost, ...items ] as any;
+		const partitionDraftPosts: Record< string, Array< Post[] > > = items.reduce(
+			( { drafts, recent }, item ) => {
+				return item.status === 'draft'
+					? { drafts: [ ...drafts, item ], recent }
+					: { drafts, recent: [ ...recent, item ] };
+			},
+			{ drafts: [] as Array< Post[] >, recent: [] as Array< Post[] > }
+		);
+
+		// @TODO Not sure if this is the most optimal model
```

Comment:
Sure, seems fine. Interesting that drafts get their own special area but other statuses (like scheduled) still use a label. But that's probably fine, users probably think of "scheduled" as a special kind published anyway.